content/hardware/03.nano/boards/nano-33-ble-sense/tutorials/get-started-with-machine-learning/get-started-with-machine-learning.md

```diff
@@ -131,10 +135,12 @@ We’ve adapted the tutorial below, so no additional hardware is needed – the
 
 ![Install Nano BLE board](assets/BoardsManager.png)
 
-**2.** Also, let's make sure we have all the libraries we need installed. If we are using the online IDE, there is no need to install anything. If we are using the offline IDE, this can be done by navigating to **Tools > Manage libraries...**, search for **Arduino_TensorFlowLite** and **Arduino:LSM9DS1**, and install them both. 
+**2.** Also, let's make sure we have all the libraries we need installed. If we are using the online IDE, there is no need to install anything. If we are using the offline IDE, this can be done by navigating to **Tools > Manage libraries...**, search for **Arduino:LSM9DS1**, and install it. 
```

Comment:
```suggestion
**2.** Also, let's make sure we have the **Arduino_LSM9DS1** library installed. If we are using the online IDE, it is pre-installed. If we are using the offline IDE, this can be done by navigating to **Tools > Manage libraries...**, search for **Arduino_LSM9DS1**, and install it. 
```

The previous wording made it sound like step (3) could be skipped if using Arduino Web Editor.

I also propose a correction of a pre-existing error in the library name (it is "Arduino_LSM9DS1", not "Arduino:LSM9DS1", and you won't get any results from the search for the incorrect name).